src/scitacean/model.py

```diff
@@ -152,6 +153,10 @@ class DownloadDataset(BaseModel):
     updatedBy: str | None = None
     validationStatus: str | None = None
 
+    @pydantic.field_validator("sourceFolder", mode="before")
+    def _validate_absolute_remote_path(cls, value: Any) -> Any:
+        return validate_absolute_remote_path(value)
+
```

Comment:
Will this mean it will fail the validation after getting a dataset that has remote path? Or it will build the dataset model anyway and just throws warnings...?

Reply:
It will currently fail when getting the dataset. I am working on relaxing this right now across all validators. So I am going to open a separate PR for it.

Reply:
See https://github.com/SciCatProject/scitacean/pull/301